ui-grid/ui-grid.d.ts

```diff
@@ -2087,6 +2087,25 @@ declare module uiGrid {
              * @default true
              */
             enableInfiniteScroll?: boolean;
+            /**
```

Comment:
It seems a little strange to me, but in the documentation for the infiniteScroll module it doesn't have these options listed in the GridOptions for the module (see http://ui-grid.info/docs/#/api/ui.grid.infiniteScroll.api:GridOptions).  Instead, I see these three options listed in the base/global module for ui-grid (see http://ui-grid.info/docs/#/api/ui.grid.class:GridOptions).  So perhaps these options should be moved to `uiGrid.IGridOptions`.  Aside from that, I'm OK with this change.